frontend/src/metabase/common/components/Table/Table.tsx

```diff
@@ -84,13 +84,17 @@ export function Table<Row extends BaseRow>({
           </tr>
         </thead>
         <tbody>
-          {rows.length > 0
-            ? rows.map((row, index) => (
-                <React.Fragment key={String(row.id) || index}>
-                  {rowRenderer(row)}
-                </React.Fragment>
-              ))
-            : emptyBody}
+          {rows.length > 0 ? (
+            rows.map((row, index) => (
+              <React.Fragment key={String(row.id) || index}>
+                {rowRenderer(row)}
+              </React.Fragment>
+            ))
+          ) : (
+            <tr className={CS.EmptyTableRow}>
+              <td colSpan={columns.length}>{emptyBody}</td>
+            </tr>
+          )}
```

Comment:
`emptyBody` is a useful prop, but it always had to be wrapped in a tr + td with a colSpan to work, this moves this wrapping into the table component itself and removes it where it is used elsewhere